repos/backend/libs/fileSys/testFiles.js

```diff
@@ -49,20 +49,31 @@ const getTestFile = async (config, testPath) => {
   } 
 }
 
+/**
+ * Save file at a given location. file should be located in the test root path
+ * @param {Object} config 
+ * @param {string} fullPath
+ * @param {string} content 
+ */
+const saveTestFile = async (config, fullPath, content) => {
 
-const saveTestFile = async (config, testPath, content) => {
   const { testsRoot } = config.paths
-  const fullPath = path.join(testsRoot, testPath)
+  const inTestRoot = fullPath.startsWith(testsRoot)
+  if (!inTestRoot) throw new Error(`[API - Files] Filepath must have rootPath '${testsRoot}': received '${fullPath}'`)
```

Comment:
Typically we don't want to expose paths to the front end for safety reasons. 
Can you update this to say something like:
"File must be saved to the mounted test folder!"

Reply:
updated